docs/information-security-policy.md

```diff
@@ -0,0 +1,70 @@
+4.  Senior Responsible Officers **SHALL** ensure that Information security be embedded into all projects and formal approval from an appropriate security representative **SHALL** be mandatory.
+5.  Senior Responsible Officers **SHALL** ensure that all information is assigned an Information Asset Owner, whose responsibilities are defined in [Guidance on the Information Asset Owner role](https://www.gov.uk/government/publications/information-asset-owner-role-guidance/guidance-on-the-information-asset-owner-role-html).
+6.  Senior Responsible Officers **SHALL** ensure effective and proportionate security controls are in place to manage any identified risks.
+7.  Senior Business Owners **SHALL** ensure the compliance of suppliers and partners with relevant MoJ security policies and understand the risk of any deviations.
+8.  Technology Service Owners **SHALL** protect technology services through security controls as detailed in MoJ policies and standards.
+9.  Technology Service Owners **SHALL** ensure that disaster recovery, and business continuity plans are implemented and tested in accordance with relevant MoJ standards.
+
+## Compliance
+
+The Security and Information Directorate \(SID\) will regularly review this policy and associated framework documents to ensure they remain suitable. SID will undertake routine compliance activities - such as reviews of security controls - against this policy and the Information Security Framework.
+
+Where requirements in this policy and the Information Security Framework have not been met and no formal risk has been escalated in-line with the relevant policies, individuals may be subject to disciplinary processes as per MoJ HR Policies. For external parties and contractors this may constitute a breach of contract. If any person is identified to have committed a criminal offence it may lead to prosecution.
+
+## Exceptions
+
+Anyone requesting a policy exemption should contact the relevant Senior Information Risk Owner \(SIRO\), with support and guidance from the MoJ Information Security Team. In the first instance, please contact the MoJ Security Team  security@justice.gov.uk to discuss mitigation options.
+
+## Related Information
+
+-   [GoVS007](https://www.gov.uk/government/publications/government-functional-standard-govs-007-security)
+-   [HMG Cyber Security Standard](https://www.security.gov.uk/policy-and-guidance/the-cyber-security-standard/)
+-   [Government Cyber Security Policy Handbook](https://www.security.gov.uk/policy-and-guidance/government-cyber-security-policy-handbook/)
+
+## Contact details
+
+For any further questions or advice relating to security, contact: [security@justice.gov.uk](mailto:security@justice.gov.uk).
+
+## Feedback
+
+> If you have any questions or comments about this guidance, such as suggestions for improvements, please contact: [itpolicycontent@digital.justice.gov.uk](mailto:itpolicycontent@digital.justice.gov.uk).
```

Comment:
We agreed to just use the mail security mailbox for Feedback, as we need to deprecate this one.